Sources/docc/DocCDocumentation.docc/adding-images.md

```diff
@@ -0,0 +1,40 @@
+# Adding Images to Your Content
+
+Elevate your content's visual appeal by adding images.
+
+## Overview
+
+DocC extends Markdown's image support so you can provide appearance and 
+display scale-aware versions of an image. You use specific components to create image filenames, and DocC uses the most appropriate version of the image when displaying your documentation.
+
+![An image of a filename that's split into four labeled sections to highlight the individual components. From left to right, the components are the image name, the appearance mode, the display scale, and the file extension.](docc-image-filename)
+
+| Component | Description |
+| --- | --- |
+| Image name | **Required**. Identifies the image within the documentation catalog. The name must be unique across all images in the catalog, even if you store them in separate folders. |
+| Appearance | **Optional**. Identifies the appearance mode in which DocC uses the image. Add `~dark` directly after the image name to identify the image as a dark mode variant. |
+| Display scale | **Optional**. Identifies the display scale at which DocC uses the image. Possible values are `@1x`, `@2x`, and `@3x`. When specifying a display scale, add it directly before the file extension. |
+| File extension | **Required**. Identifies the type of image, such as .png or .jpeg. |
```

Comment:
Are there any unsupported image file types for images? For example, what about `.gif`, `.tiff`, `.svg`, et al.?

Reply:
The current supported image types are "png", "jpg", "jpeg", "svg" and "gif" IMO.

https://github.com/apple/swift-docc/blob/38c9b8abc98ff7227a1b5656ebc29684c2a459ef/Sources/SwiftDocC/Infrastructure/DocumentationContext.swift#L1722-L1723

If you are interested adding more supported image type, you can file a issue or create a PR to fix it. (eg. #4 add supports for "gif" support)

Reply:
Thank Kyle. Resolved here https://github.com/apple/swift-docc/pull/742/commits/ec67b2c642577028a3973f18087d7b22a936a8ff